native_client/java/libdeepspeech/src/androidTest/java/org/mozilla/deepspeech/libdeepspeech/test/BasicTest.java

```diff
@@ -149,4 +151,36 @@ public void loadDeepSpeech_sttWithMetadata_withLM() {
         assertEquals("she had your dark suit in greasy wash water all year", decoded);
         m.freeModel();
     }
+
+    @Test
+    public void loadDeepSpeech_HotWord_withLM() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+        m.enableExternalScorer(scorerFile);
+
+        for(int i = 0; i < word.length; i++) {
+            m.addHotWord(word[i], boost[i]);
+            String decoded = doSTT(m, false);
+            assertEquals("she had your dark suit in greasy wash water all year", decoded);
+            m.eraseHotWord(word[i]);
+        }
+
+        m.freeModel();
+    }
+
+    @Test
+    public void loadDeepSpeech_HotWord_noLM() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+        try {
+            m.addHotWord(word[0], boost[0]);
+            assert(false);
+        }
+        catch(Exception e) {
+            assertEquals("Error: External scorer is not enabled. (0x2004).", e.getMessage());
+        }
+        finally {
+            m.eraseHotWord(word[0]);
```

Comment:
Remove this

Reply:
Do u want me to remove the finally block or the entire test?

Reply:
just the `m.eraseHotWord` call